mne_bids/tests/test_write.py

```diff
@@ -10,6 +10,7 @@
 #          Matt Sanderson <matt.sanderson@mq.edu.au>
 #
 # License: BSD (3-clause)
+from mne_bids.write import mark_bad_channels
```

Comment:
wrong import order.

local package imports should be last

Reply:
Thanks, this was actually caused by Pylance automatically adding an import. Need to turn off this feature!

Reply:
Fixed in c5f997c